src/test/ui/suggestions/constrain-trait.stderr

```diff
@@ -0,0 +1,27 @@
+error[E0599]: no method named `get_a` found for type `&Self` in the current scope
+  --> $DIR/constrain-trait.rs:15:31
+   |
+LL |         println!("{:?}", self.get_a());
+   |                               ^^^^^ method not found in `&Self`
+   |
+   = help: items from traits can only be used if the type parameter is bounded by the trait
+help: the following trait defines an item `get_a`, perhaps you need to restrict type parameter `Self` with it:
```

Comment:
This wording feels ungreat -- I would wonder where the hell the type parameter `Self` came from since it's not in the source. I would say *", perhaps you need to add another super-trait:"*